macos/pkg/CMakeLists.txt

```diff
@@ -51,7 +51,7 @@ add_custom_command(
     DEPENDS mozillavpn
     COMMAND ${CMAKE_COMMAND} -E remove_directory -f ${CMAKE_CURRENT_BINARY_DIR}/staging
     COMMAND ${CMAKE_COMMAND} -E make_directory ${CMAKE_CURRENT_BINARY_DIR}/staging
-    COMMAND ${CMAKE_COMMAND} -E copy_directory $<TARGET_BUNDLE_DIR:mozillavpn>
+    COMMAND ${CMAKE_COMMAND} -E copy_directory $<TARGET_FILE_DIR:mozillavpn>
```

Comment:
Yeap, it's unrelated, however without this change I was not able to work on the changes I made here and it was so small I decided to keep it. However, I am not too sure about it. From the commit message:

> My experience with CMake is none, so I am not really sure what was going
on here. What I know is that I was getting the following error:
> 
> TARGET_BUNDLE_DIR is allowed only for Bundle targets
> 
> And with the change in this commit I don't get it anymore. Added this in
a separate commit so that it can be easily dropped if this was just me
doing something wrong.

Reply:
https://github.com/mozilla-mobile/mozilla-vpn-client/pull/4030